### PR TITLE
Added menu and action bar interactions

### PR DESCRIPTION
**Pull request by alancamargo92**
### New features
* Interactions with options menu
* Interactions with action bar

### Sample code
* Created activity with options menu and action bar

### Tests
* Added 3 tests to validate new features
* All tests passing, except for 2 which are already failing in develop

### CODE REVIEW

File: kappuccino/src/main/kotlin/br/com/concretesolutions/kappuccino/custom/menu/MenuMethods.kt [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+}
+
+class MenuMethods {
+
+    fun openOptionsMenu(): MenuMethods {
+        openActionBarOverflowOrOptionsMenu(getTargetContext())
+        return this
+    }
+
+    fun onItem(@StringRes itemTextId: Int, func: Interactions.() -> Interactions): MenuMethods {
```
Comment (heitorcolangelo) [inline kappuccino/src/main/kotlin/br/com/concretesolutions/kappuccino/custom/menu/MenuMethods.kt]: Why did you decided to match the views using `withText()`?
Maybe it's a good idea to let the user match the view with some other attribute like the `Id`.

Comment (alancamargo92) [reply kappuccino/src/main/kotlin/br/com/concretesolutions/kappuccino/custom/menu/MenuMethods.kt]: Unfortunately options menu items are only matched with their text (I tried with the ID and it didn't work), only action bar items seem to work that way

Comment (heitorcolangelo) [reply kappuccino/src/main/kotlin/br/com/concretesolutions/kappuccino/custom/menu/MenuMethods.kt]: This is really weird! haha but I think that there's not much to do then... 🤷🏽‍♂️

Comment (alancamargo92) [reply kappuccino/src/main/kotlin/br/com/concretesolutions/kappuccino/custom/menu/MenuMethods.kt]: Yeah I know =/

**Comment by alancamargo92** [comment]
@cs-bruno-silva what do you think?

**Pull request merged**